### PR TITLE
Fix forced reference to api path

### PR DESCRIPTION
**Pull request by stlewis**
I think it's overly restrictive to force documentation into a `docs/api` directory. This makes sense in contexts where an API lives alongside other aspects of an application, (where you would expect API controllers to be name-spaced in this fashion), but what about API-only projects where having `api` anywhere in the route would be redundant?

This issue is even larger when you consider what happens when you run the documentation generator. It uses the path as-provided by the user:

`rails g apidoco v1/customers`

The line above currently creates the directory `docs/v1/customers/`, which is exactly what you would expect, but then when you attempt to view the documentation, everything blows up because the `VersionParser` is forcibly prepending `api` to whatever the user provides.

This PR fixes that by simply not adding the `api` part and relying upon user input to do so.

### CODE REVIEW

File: spec/dummy/log/test.log [hunk 1]
```diff
@@ -3,3 +3,148 @@
...
+Processing by Apidoco::ApisController#show as HTML
+  Parameters: {"id"=>"api"}
+  Rendering /Users/stevelewis/Projects/gems/apidoco/app/views/apidoco/apis/show.html.erb within layouts/apidoco/application
+  Rendered /Users/stevelewis/Projects/gems/apidoco/app/views/apidoco/apis/_side_bar.html.erb (0.0ms)
+  Rendered /Users/stevelewis/Projects/gems/apidoco/app/views/apidoco/apis/_examples.html (0.0ms)
+  Rendered /Users/stevelewis/Projects/gems/apidoco/app/views/apidoco/apis/show.html.erb within layouts/apidoco/application (0.6ms)
+Completed 200 OK in 2ms (Views: 1.6ms | ActiveRecord: 0.0ms)
+Started GET "/assets/apidoco/application-2c1ae9b956698a27fcf235f867cbca704117db2e7fafb5ef3e46dd5bf76166ab.css" for 127.0.0.1 at 2020-05-18 11:31:43 -0700
+Started GET "/assets/apidoco/application-2a74775aedd30b28e332bc8f060e37ac2fa9939847147d8c4ee7b62bf07ff8ad.js" for 127.0.0.1 at 2020-05-18 11:31:43 -0700
+  [1m[35m (0.1ms)[0m  [1m[31mrollback transaction[0m
```
Comment (alameenkhader) [inline spec/dummy/log/test.log:150]: We should not check in the log files and the sqlite3.

**Comment by stlewis** [comment]
@alameenkhader My mistake, sorry about that! Fix is in place. By the way, I hate to just come in suggesting fixes without saying that this is an awesome gem that is _exactly_ what I've been looking for to document my project! :-D

**Comment by alameenkhader** [comment]
@stlewis I will do a manual test and merge it at the earliest. Thanks for the PR.

You are always welcome with PRs to improve this project.

**Comment by alameenkhader** [comment]
@stlewis This is not working as expected for my folder structure.
image
I think we should have a default documentation path to look for and I prefer that to be `doc/api/` for the sake of existing users of this gem and we can have the documentation path configurable. 

We need to make the base path configurable in the initiaizer.
```ruby
Apidoco.app_name = ''Apidoco Demo"
Apidoco.base_path = "docs"
```
and use the configured basepath if defined in the `VersionParser`. 

Let me know if you have any questions. Thanks!

**Comment by stlewis** [comment]
@alameenkhader I like the idea of a configurable base path. I'll get an update on the PR shortly.

**Comment by stlewis** [comment]
@alameenkhader Changes made to support a configurable path.

**Pull request merged**